tutorials/introductory/usage.py

```diff
@@ -121,8 +121,8 @@
 # =====================================
 #
 # All of plotting functions expect `numpy.array` or `numpy.ma.masked_array` as
-# input.  Classes that are 'array-like' such as `pandas` data objects
-# and `numpy.matrix` may or may not work as intended.  It is best to
+# input.  Classes that are similar to arrays such as `pandas` data objects
```

Comment:
you may want to include (`array-like`) in parenthesis as that's the numpydoc standard and so shows up all over the doc strings.